packages/connect-webextension/src/index.ts

```diff
@@ -18,6 +18,7 @@ import { initLog, setLogWriter, LogMessage, LogWriter } from '@trezor/connect/li
 
 import * as popup from './popup';
 import { parseConnectSettings } from './connectSettings';
+import { ServiceWorkerWindowChannel } from './channels/serviceworker-window';
 
 const eventEmitter = new EventEmitter();
```

Comment:
it is event emitter for compatibility but in fact, at least some of the events if not all of them never arrive. I would consider signaling this to implementer somehow. maybe by overriding `.on` method. but this can be done later.